src/sentry/api/endpoints/project_user_issue.py

```diff
@@ -0,0 +1,239 @@
+from datetime import UTC, datetime
```

Comment:
can we move this file to `src/sentry/issues/endpoints` to live with the rest of the issues endpoints? 🙏🏼